solutions/landing-zone-v2/setters.yaml

```diff
@@ -45,11 +45,9 @@ data:
   # Cannot end with a hyphen.
   # Cannot be in use or previously used; this includes deleted projects.
   # Cannot contain restricted strings, such as google and ssl.
-  net-host-prj-nonprod-id: net-host-prj-nonprod-12345
-  net-host-prj-prod-id: net-host-prj-prod-12345
-  net-perimeter-prj-common-id: net-per-prj-common-12345
   audit-prj-id: audit-prj-id-12345
   guardrails-project-id: guardrails-project-12345
+  hub-project-id: hub-project-12345
```

Comment:
is the `hub-project-id` setter required in the landing-zone-v2 package, it doesn't seem to be used outside of the `hub-env` package?

maybe for future use?

Reply:
It is being used in the spoke project to define with which network the vpc peering should be established. I say we leave it there for now and re-validate as we work on the spoke project